### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by charshy**
- Fixed step error (codeblock steps not matching code)
- Tweaked grammar and explanations for clarity
- Standardized use of terminology, e.g. demo -> sample.

### CODE REVIEW

File: apps/helloworld/README.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ Before you begin, follow the environment set up instructions at https://github.c
     ...
     ```
 
-1. Next, we will split traffic between helloworld v1 and v2
+Next, split traffic so thart it is routed between both helloworld v1 and helloworld v2.
```
Comment (frankbu) [inline apps/helloworld/README.md]: typo:  so thart it is routed

**Comment by frankbu** [comment]
The step numbers are screwed up with this patch. Note that you can't set the starting number of items in a markdown list

**Comment by charshy** [comment]
I've updated the file and committed to this patch to resolve the step numbering and typo as discussed. I've used an external markdown editor to assist - please let me know if the line number problems persist.

**Comment by frankbu** [comment]
The line numbers of the various steps are still all screwed up in this patch - keeps going back to step 1. Is there another version that supersedes this pull request?

**Pull request closed without merging**